tests/acceptance/conftest.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import
+
+import os
+import json
+import subprocess
+import time
+
+
+def pytest_configure(config):
+    """
+    Generate frontend assets before running any acceptance tests
+
+    TODO: There is a bug if you run `py.test` with `-f` -- the built
+    assets will trigger another `py.test` run.
+    """
+
+    # Do not build in CI because tests are run w/ `make test-acceptance` which builds assets
+    if os.environ.get("CI") or os.environ.get("SKIP_UI_BUILD"):
```

Comment:
What's this magic `CI` env var?

Reply:
It's one of Travis's default env vars

Reply:
Can you add a/edit the comment mentioning it's travis related? Main reason being if we move off travis in the future, it'll be easy to grep for.